src/fr-webauthn/index.ts

```diff
@@ -235,15 +259,13 @@ abstract class FRWebAuthn {
    * @return The outcome string
    */
   public static getAuthenticationOutcome(credential: PublicKeyCredential | null): string {
-    if (!window.PublicKeyCredential) {
-      return WebAuthnOutcome.Unsupported;
```

Comment:
I found it odd that the feature check was done after the API was accessed. So, I moved it to just before the API was used on line 246 above.